posthog/queries/actor_base_query.py

```diff
@@ -196,6 +197,7 @@ def serialize_people(data: QuerySet[Person]) -> List[SerializedPerson]:
         SerializedPerson(
             type="person",
             id=person.uuid,
+            uuid=person.uuid,
```

Comment:
doesn't seem enough, I think you need to update L189 above to include `uuid` as well?

Reply:
oh weird, it was working with `id = person.uuid` ? 🤔

Reply:
The test is passing so... I think it's fine?

I've just moved it to just the PersonSerializer to avoid changes to the groups tests 🤞 

There seemed to also be an automated snapshot change which I don't think is correct, so I've force pushed that away :\

Reply:
oh, that test is annoying, feel free to remove the snapshots. It's happening because it's order of execution dependent - newer flags get a higher ID, which is destroying the snapshot 😮‍💨

Reply:
lol, just looked at L189 again, and in the end, it has 'uuid', which I glossed over 🤦 .

Also: `only` is only for deferring, not that it's not selected at all

https://docs.djangoproject.com/en/4.0/ref/models/querysets/#django.db.models.query.QuerySet.only

D'oh.